howso/react_aggregate.amlg

```diff
@@ -760,4 +849,84 @@
 			))
 		)
 	)
+
+	;Helper method for ReactAggregate that checks the output to see if the requested stat exists
+	#!ValidateCachedStats
+	(declare
+		(assoc
+			features (list)
+			stat_name (null)
+			selected_prediction_stats (null)
+		)
+
+		(declare (assoc exists (true)))
+
+		;prediction stats validation depends on the selected prediction stats
+		(if (= stat_name "prediction_stats")
+			(if
+				;if mae is the only prediction stat only, make sure every feature has a mae value
+				(and
+					(= (size selected_prediction_stats) 1)
+					(= (first selected_prediction_stats) "mae")
+				)
+				(seq
+					(map
+						(lambda
+							(if
+								(not (get output ["payload" (current_value 1) "mae"]))
+								(assign (assoc exists (false)))
+							)
+						)
+						features
+					)
+					exists
+				)
+				; else if it includes other prediction stats, check to see if any exists. If any
+				;  exists, then it means that they all should exist. This is checked this way since
+				;  prediction stats varies by feature type (nominal vs continuous).
+				(seq
+					(assign (assoc
+						selected_prediction_stats
+							(filter
+								;exclude mae, as mae can be returned even if the other prediction stats are not calculated
+								(lambda (not (contains_value ["mae"] (current_value))))
```

Comment:
`(not (contains_value ["mae"] (current_value)))` should be `(!= "mae" (current_value))`